cli/src/main/java/com/liferay/blade/cli/util/BladeUtil.java

```diff
@@ -563,6 +536,54 @@ public static void unzip(File srcFile, File destDir, String entryToStart) throws
 		}
 	}
 
+	public static void unzipStream(InputStream in, File outputDir) throws Exception {
+		unzipStream(in, outputDir, true);
+	}
+
+	public static void unzipStream(InputStream in, File outputDir, boolean preserveNewerFiles) throws Exception {
```

Comment:
Can you use the zip-slip.zip tests zip that is checked into our repo to make sure that this method will handle it correctly?